src/views/Admin/AdminTransfer.js

```diff
@@ -47,46 +53,64 @@ export default function AdminTransfer() {
     bind,
   } = useTransfer();
 
-  const [addressInput, { pass, data: address }] = useAddressInput({
-    name: 'address',
-    label: `Ethereum Address`,
-    disabled: inputsLocked,
-  });
+  const validate = useMemo(
+    () => composeValidator({ address: buildAddressValidator() }),
+    []
+  );
 
-  useEffect(() => {
-    if (pass) {
-      construct(address);
-    } else {
-      unconstruct();
-    }
-  }, [pass, address, construct, unconstruct]);
+  const onValues = useCallback(
+    ({ valid, values }) => {
+      if (valid) {
+        construct(values.address);
+      } else {
+        unconstruct();
+      }
+    },
+    [construct, unconstruct]
+  );
 
   return (
     <Grid>
       <Grid.Item full as={ViewHeader}>
         Transfer Point
       </Grid.Item>
-      <Grid.Item
-        full
-        as={Text}
-        className={cn('f5', {
-          green3: completed,
-        })}>
-        {completed
-          ? `${address} is now the Transfer Proxy for ${name} and can accept the transfer by logging into Multipass themselves. Until they accept your transfer, you will still have ownership over ${name}.`
-          : `Transfer ${name} to a new owner.`}
-      </Grid.Item>
 
-      {!completed && (
-        <Grid.Item full as={Input} {...addressInput} className="mv4" />
-      )}
+      <BridgeForm validate={validate} onSubmit={() => {}} onValues={onValues}>
+        {({ handleSubmit, values }) => (
+          <>
+            <Grid.Item
+              full
+              as={Text}
+              className={cn('f5', {
+                green3: completed,
+              })}>
+              {completed
+                ? `${values.address} is now the Transfer Proxy for ${name} and can accept the transfer by logging into Multipass themselves. Until they accept your transfer, you will still have ownership over ${name}.`
```

Comment:
s/Multipass/Bridge again. You're probably fine to pull in mino now that #219 is merged and find-replace the remaining instances.

Reply:
merged mino & fixed 👍